src/main/resources/static/js/subscription.js

```diff
@@ -1090,7 +1088,8 @@ jQuery(document).ready(function () {
                 var errors = "";
                 for (var i = 0; i < responseJSON.length; i++) {
                     errors = errors + "\n" + responseJSON[i].reason;
-                    $.jGrowl(responseJSON[i].subscription + " :: " + responseJSON[i].reason, { sticky: true, theme: 'Error' });
+                    // Errors are displayed in the form.
+                    // $.jGrowl(responseJSON[i].subscription + " :: " + responseJSON[i].reason, { sticky: true, theme: 'Error' });
```

Comment:
Remove this commented out code if it is not going to be used.

Reply:
Fixed.